[2022-02-15 03:15:02,863] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:02,904] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:15:02,905] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:02,907] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:15:02,908] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:15:02,936] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.fct_orders> on 2022-02-15 03:14:06.780838+00:00
[2022-02-15 03:15:02,944] {standard_task_runner.py:52} INFO - Started process 1292 to run task
[2022-02-15 03:15:02,975] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.fct_orders', 'manual__2022-02-15T03:14:06.780838+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpbco1km23', '--error-file', '/tmp/tmp37uh8z1v']
[2022-02-15 03:15:02,976] {standard_task_runner.py:77} INFO - Job 113: Subtask model.my_new_project.fct_orders
[2022-02-15 03:15:03,198] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:14:06.780838+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:15:03,445] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.fct_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:14:06.780838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:14:06.780838+00:00
[2022-02-15 03:15:03,447] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:15:03,449] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models fct_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:15:03,481] {subprocess.py:85} INFO - Output:
[2022-02-15 03:15:16,073] {subprocess.py:89} INFO - 03:15:16  Running with dbt=1.0.1
[2022-02-15 03:15:16,719] {subprocess.py:89} INFO - 03:15:16  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:15:16,727] {subprocess.py:89} INFO - 03:15:16
[2022-02-15 03:15:21,090] {subprocess.py:89} INFO - 03:15:21  Concurrency: 4 threads (target='dev')
[2022-02-15 03:15:21,093] {subprocess.py:89} INFO - 03:15:21
[2022-02-15 03:15:21,128] {subprocess.py:89} INFO - 03:15:21  1 of 1 START table model dbt_bytecode_training.fct_orders....................... [RUN]
[2022-02-15 03:15:24,664] {subprocess.py:89} INFO - 03:15:24  1 of 1 OK created table model dbt_bytecode_training.fct_orders.................. [SUCCESS 1 in 3.53s]
[2022-02-15 03:15:24,678] {subprocess.py:89} INFO - 03:15:24
[2022-02-15 03:15:24,680] {subprocess.py:89} INFO - 03:15:24  Finished running 1 table model in 7.95s.
[2022-02-15 03:15:24,685] {subprocess.py:89} INFO - 03:15:24
[2022-02-15 03:15:24,687] {subprocess.py:89} INFO - 03:15:24  Completed successfully
[2022-02-15 03:15:24,689] {subprocess.py:89} INFO - 03:15:24
[2022-02-15 03:15:24,691] {subprocess.py:89} INFO - 03:15:24  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:15:25,112] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:15:25,144] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.fct_orders, execution_date=20220215T031406, start_date=20220215T031502, end_date=20220215T031525
[2022-02-15 03:15:25,198] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:15:25,291] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
